pyproject.toml

```diff
@@ -0,0 +1,11 @@
+[build-system]
+requires = [
+    "numpy>=1.15,<1.16; python_version=='3.6'",
```

Comment:
Do you think we need to include pybind11? It's a requirement in setup.py, definitely needed to build.

Reply:
how about threadpoolctl? is it needed for build too?

Reply:
I'm not sure. I'm not the package author. My guess would be no. @akshayka @sbarratt do you know if threadpoolctl is a build requirement?

Reply:
No, threadpoolctl shouldn't be needed by the build system (it's just an installation dependency, not a build dependency).